### ISSUE TITLE
RBAC Plugin

### ISSUE THREAD

**Issue by altvnk** [CiscoCloud/kubernetes-ansible#60]
[Kismatic](https://kismatic.io) Kubernetes role-based access control (RBAC) [plug-in](https://github.com/kismatic/kubernetes-rbac)

**Comment by cdmitri** [comment]
Yes :) these guys are cool, talked to them at mesoscon. Check it out and
we'll meet with them...also need it for some frameworks

**Comment by peterlamar** [comment]
+1
